content/_data/authors/timja.adoc

```diff
@@ -2,3 +2,6 @@
 name: "Tim Jacomb"
 github: "timja"
 ---
+Jenkins core maintainer, along with slack, azure-keyvault and configuration-as-code plugins.
+Tim started using Jenkins in 2013 and became an active contributor in 2018.
+Tim enjoys working on open source software in his “free” time.
```

Comment:
Consider using simpler double quotes rather than smart quotes?  I'd rather not discover surprises related to processing smart quotes.
```suggestion
Tim enjoys working on open source software in his "free" time.
```

Reply:
This is 2020, Unicode works fine…